package.json

```diff
@@ -1,14 +1,18 @@
 {
   "private": true,
-  "devDependencies": {
-    "gulp": "^3.8.8"
+  "scripts": {
+    "prod": "gulp --production",
+    "dev": "gulp watch"
   },
-  "dependencies": {
-    "bootstrap-sass": "^3.0.0",
-    "laravel-elixir": "^4.0.0",
+  "devDependencies": {
+    "bootstrap-sass": "^3.3.7",
+    "gulp": "^3.9.1",
+    "laravel-elixir": "^6.0.0-14",
     "laravel-elixir-vueify": "^1.0.2",
-    "sweetalert": "^1.1.3",
     "underscore": "^1.8.3",
     "vue-resource": "^0.7.0"
+  },
+  "dependencies": {
```

Comment:
@mattstauffer Good question. I was trying to make it more in sync with what Laravel 5.3 had, and it seemed to work fine, but I think the safest bet is to revert it to what it was originally since we know that works. And if we want to clean it up more later it could be a part of a larger effort to move things over to Laravel Mix, etc.